sematic/future.py

```diff
@@ -89,9 +95,15 @@ def set(self, **kwargs):
                 )
 
         if "inline" in kwargs:
+            warn(INLINE_DEPRECATION_MESSAGE, DeprecationWarning)
```

Comment:
Can you please update the PR description with the produced logs?

[The documentation](https://docs.python.org/3/library/warnings.html) states that by default `DeprecationWarning` is not printed:
> Base category for warnings about deprecated features when those warnings are intended for other Python developers (ignored by default, unless triggered by code in __main__).